### PR TITLE
Search UI

### LINKED ISSUE #178: Strange clipping of Login bubbles
There is some weird clipping effect that is appearing in the top left of the login page near the "Logout" button

simulator_screenshot_E9C65ED2-6876-4F08-B16A-7694BD10579A

### PR DESCRIPTION
**Pull request by spencerdearman**
- Working with all new colors
- Fixing the geometry of a bunch of pages
- Changing the search UI

### CODE REVIEW

File: DiveMeets/DiveMeets/Resources/Colors.swift [hunk 1]
```diff
@@ -21,4 +21,9 @@ struct Custom {
     static let selection = Color("Selection")
     static let textColor = Color("TextColors")
     static let secondaryColor = Color("SecondaryColor")
+    static let grayThinMaterial = Color("GrayThinMaterial")
+    static let specialGray = Color("specialGray")
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Resources/Colors.swift]:
```suggestion
    static let specialGray = Color("SpecialGray")
```

File: DiveMeets/DiveMeets/Views/Login/LoginProfile.swift [hunk 2]
```diff
@@ -35,7 +35,10 @@ struct LoginProfile: View {
     @Binding var loginSearchSubmitted: Bool
     @State var diverData : [[String]] = []
     @State var profileType : String = ""
+    @State var isLoading: Bool = true
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Login/LoginProfile.swift]: Is this being used?

File: DiveMeets/DiveMeets/Views/Main/ContentView.swift [hunk 3]
```diff
@@ -66,13 +66,13 @@ struct ContentView: View {
                                     case .house:
                                         Home()
                                     case .wrench:
-//                                        NavigationView {
-//                                            LiveResultsView(request: "debug")
+                                        NavigationView {
+                                            LiveResultsView(request: "debug")
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/ContentView.swift]:
```suggestion
                                        //NavigationView {
                                            //LiveResultsView(request: "debug")
```

File: DiveMeets/DiveMeets/Views/Main/ContentView.swift [hunk 4]
```diff
@@ -66,13 +66,13 @@ struct ContentView: View {
...
 //                                            FinishedLiveResultsView(link: "https://secure.meetcontrol.com/divemeets/system/livestats.php?event=stats-9050-770-9-Finished")
-//                                        }
-//                                        .navigationViewStyle(StackNavigationViewStyle())
-                                        ToolsMenu()
-                                        //MeetDBTestView()
+                                        }
+                                        .navigationViewStyle(StackNavigationViewStyle())
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/ContentView.swift]:
```suggestion
                                        //}
                                        //.navigationViewStyle(StackNavigationViewStyle())
```

File: DiveMeets/DiveMeets/Views/Main/ContentView.swift [hunk 5]
```diff
@@ -66,13 +66,13 @@ struct ContentView: View {
...
+                                        //ToolsMenu()
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/ContentView.swift]:
```suggestion
                                        ToolsMenu()
```

File: DiveMeets/DiveMeets/Views/Search/RecordList.swift [hunk 6]
```diff
@@ -43,7 +43,7 @@ struct RecordList: View {
     var body: some View {
         ZStack {
             // Background color for View
-            customGray.ignoresSafeArea()
+            Custom.specialGray
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Search/RecordList.swift:46]: Does this need to ignore the safe area?

**Review by lsherwin10 — CHANGES_REQUESTED** [review]
Just a couple small things

File: DiveMeets/DiveMeets/Views/Login/LoginProfile.swift (hunk 2 above)
Comment (spencerdearman) [reply DiveMeets/DiveMeets/Views/Login/LoginProfile.swift]: Oh no, that was from something else, I will remove

File: DiveMeets/DiveMeets/Views/Search/RecordList.swift (hunk 6 above)
Comment (spencerdearman) [reply DiveMeets/DiveMeets/Views/Search/RecordList.swift:46]: I do not think so, the change is good

File: DiveMeets/DiveMeets/Views/Login/LoginProfile.swift [hunk 7]
```diff
@@ -68,9 +70,6 @@ struct LoginProfile: View {
         
         if profileType == "Diver" {
             ZStack{
-                GeometryReader{ geometry in
-                    BackgroundSpheres()
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Login/LoginProfile.swift:58]: Are the BackgroundSpheres supposed to be removed here, or just the GeometryReader?

File: DiveMeets/DiveMeets/Views/Main/ContentView.swift [hunk 8]
```diff
@@ -12,7 +12,7 @@ struct ContentView: View {
     @Environment(\.scenePhase) var scenePhase
     @Environment(\.meetsDB) var db
     @EnvironmentObject private var p: MeetParser
-    @State private var selectedTab: Tab = .house
+    @State private var selectedTab: Tab = .wrench
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/ContentView.swift]:
```suggestion
    @State private var selectedTab: Tab = .house
```

**Review by lsherwin10 — COMMENTED** [review]
Two other things, my bad I didn't see them before

File: DiveMeets/DiveMeets/Views/Login/LoginProfile.swift (hunk 7 above)
Comment (spencerdearman) [reply DiveMeets/DiveMeets/Views/Login/LoginProfile.swift:58]: Both, that was the reason for the other issue with the weird glitch in the profile

**Review by lsherwin10 — APPROVED** [review]
Perfect

**Pull request merged**